alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/destination/Destination.kt

```diff
@@ -307,4 +315,12 @@ data class Destination(
         }
         return content
     }
+
+    private fun validateDestinationUri(destinationMessage: BaseMessage, denyHostRanges: List<String>) {
+        if (DestinationType.EMAIL.equals(destinationMessage.channelType)) {
```

Comment:
Any reason this is only done for `EMAIL`? `isHostInDenylist` seems to be implemented generically enough to catch any denied `hosts` if set for other `DestinationType`s

Reply:
I think this should be only checking the custom webhook destination type. Please add integ tests to catch this type of mistake